extensions/rvadapter/src/main/java/com/google/android/agera/rvadapter/RepositoryPresenterCompilerStates.java

```diff
@@ -136,5 +164,13 @@
      */
     @NonNull
     RPItemCompile<T> stableId(long stableId);
+
+    /**
+     * Specifies a {@link Binder} to bind a single item in the {@link Repository} to an inflated
```

Comment:
... to bind the whole collection (the repository value) to an inflated item view. This binder will be called before the item binder specified with `{@link #bindWith}`, and called for each item view. This is useful for cases where the collection object contains useful information for binding items, which is not copied to the individual item objects.